lib/eventmesh/Etcd3EventMeshServer.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+const config = require('../config');
+const logger = require('../logger');
+const CONST = require('../../lib/constants');
+const EventMeshServer = require('./EventMeshServer');
+const {
+  Etcd3
+} = require('etcd3');
+
+const etcd = new Etcd3({
+  hosts: config.etcd.url[0]
+});
+
+class Etcd3EventMeshServer extends EventMeshServer {
```

Comment:
Would make sense to outline the properties/attributes of a resource and an annotation as comment. Either here or the super class.

Reply:
Added at [diff-link](https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/264/files#diff-16bc3b001161ff904934a6b2aa09415aR11)